src/ralph/dhcp/views.py

```diff
@@ -136,11 +161,25 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class DHCPNetworksView(DHCPConfigMixin, LastModifiedMixin, TemplateView):
+class DHCPNetworksView(
+    DHCPConfigMixin, LastModifiedMixin, TemplateView, APIView
+):
     template_name = 'dhcp/networks.conf'
 
     def get_last_modified(self, networks):
-        return self.last(networks)
+        last_items = []
+        last = partial(self.last, last_items=last_items)
+        last(networks)
```

Comment:
Like you noticed, this could be explicit assignment + https://github.com/ar4s/ralph/blob/ng-dhcp-agent/src/ralph/dhcp/views.py#L31